qa/tasks/cephfs/caps_helper.py

```diff
@@ -90,9 +88,23 @@ def _gen_mds_cap_str(perm, fsname=None, cephfs_mntpt='/'):
     return mds_cap
 
 
-# XXX: Inheriting CephFSTestCase ONLY to get methods like self.assertEqual(),
-# self.assertIn(), etc.
-class CapTester(CephFSTestCase):
+def assert_equal(first, second):
+    msg = f'Variables are not equal.\nfirst = {first}\nsecond = {second}'
+    assert first == second, msg
+
+
+def assert_in(member, container):
+    msg = dedent(f'''
+        First value is absent in second value.
+        First value -
+        {member}
+        Second value -
+        {container}
+        ''')
+    assert member in container, msg
+
+
+class CapTester:
```

Comment:
You plan to fold this into commit 249d6fed08cb6209f5173afb9c915fbe47be4934?

Reply:
Let's keep it separate? That commit already has 2 sets of changes.

Reply:
Logically both are different set of changes.

Also, you are mistaken. The change was not introduced by commit https://github.com/ceph/ceph/commit/249d6fed08cb6209f5173afb9c915fbe47be4934. CapTester inherits CephFSTestCase also in main branch.

Reply:
Oh. so you just added a comment saying "inherits CephFSTestCase for assertEquals" :)

Reply:
Anyway, now I'm much much more inclined in not inheriting from CephFSTestCase just to get one silly helper.